src/Simulation/PassRate.jl

```diff
@@ -0,0 +1,46 @@
+# Predicts whether a student will pass a course using the course's passrate
+# as a probability.
+
+module PassRate
```

Comment:
Snake case variables and functions

Reply:
Done. 179c41190af4e3708a1c12d1494416deb9d7eb74